util/config/update-if-different

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
```

Comment:
Have you checked to see if this script is python 2 and 3 compatible?

I'm surprised how much code is here for something so simple. It seems like this could be simple bash with something like:

```sh
dst="$1"
src="$2"
if cmp -s "$src" "$dst" ; then
   rm "$src"
else
   mv "$src" "dst"
fi
```

Or if we want to keep python this could be a lot smaller with:

```python
#!/usr/bin/env python

import filecmp
import optparse
import os
import shutil
import sys

parser = optparse.OptionParser(
    usage = 'usage: %prog dst src',
    description = 'Updates dst with src. If they differ, copies dst to src. '
                  'Deletes src either way.')

(options, args) = parser.parse_args()

if len(args) != 2:
    parser.error("Requires exactly 2 arguments")

(dst, src) = args

if not os.path.exists(dst) or not filecmp.cmp(src, dst):
    sys.stdout.write("Copying {0} to {1}\n".format(src, dst))
    shutil.copyfile(src, dst)

os.remove(src)
```

I guess I'm ok keeping what's here if you don't want to update, but it took me a while to parse what this code was doing and it reimplements things in the standard library.